Shoko.Server/Renamer/RenameFileHelper.cs

```diff
@@ -78,11 +78,8 @@ public static string GetFilename(SVR_VideoLocal_Place place, string scriptName)
                 }
                 catch (Exception e)
                 {
-                    Logger.Error($"Renamer threw an error. The offending plugin was: {renamer.GetType().GetAssemblyName()} with renamer: {renamer.GetType().Name}. The error was: {e}");
```

Comment:
We shouldn't be losing the logging detail, so I would like something back, even if it was in the `continue` case